authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+```
+
+Add Bob and an initial balance to the genesis file:
+```
+simd add-genesis-account bob 5000000000stake
+```
+
+Generate a transaction adding Alice to the initial validator set:
+```
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+```
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+```
+simd start
+```
+
+## Submit Proposal
+
+To demonstrate an authorization to vote on a governance proposal, you must first create a governance proposal.
+
+Create proposal:
+```
+simd tx gov submit-proposal --title="Test Authorization" --description="Is Bob authorized to vote?" --type="Text" --deposit="10000000stake" --from alice
+```
+
+View proposal:
+```
+simd q gov proposal 1
+```
+
+## Grant Authorization
+
+Next, the granter must `grant` an authorization to the grantee.
+
+The authorization is a "generic" authorization, meaning there are no limits on the authorization. In this case, the grantee is able to vote as many times as they want until the granter revokes the authorization.
+
+Create authorization:
+```
+simd tx authz grant cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt generic --msg-type /cosmos.gov.v1beta1.MsgVote --from alice
+```
+
+View authorization:
+```
+simd q authz grants cosmos1jxd2uhx0j6e59306jq3jfqs7rhs7cnhvey4lqh cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt /cosmos.gov.v1beta1.MsgVote
+```
+
+## Generate Transaction
+
+In order for the grantee to execute a message on behalf of the granter, the grantee must first generate an unsigned transaction where the transaction author (the `--from` address) is the granter. 
+
+Create unsigned transaction:
+```
+simd tx gov vote 1 yes --from cosmos1jxd2uhx0j6e59306jq3jfqs7rhs7cnhvey4lqh --generate-only > tx.json
+```
+
+View transaction:
+```
+cat tx.json
+```
+
+## Execute Transaction
+
+Finally, the grantee can sign and send the transaction using the `exec` command. The author of the transaction (the `--from` address) is the grantee.
+
+Sign and send transaction:
+```
+simd tx authz exec tx.json --from bob
+```
+
+View vote:
+```
+simd q gov vote 1 cosmos1jxd2uhx0j6e59306jq3jfqs7rhs7cnhvey4lqh
+```
+
+## Revoke Authorization
+
+The granter can revoke the authorization from the grantee using the `revoke` command.
+
+Revoke authorization:
+```
+simd tx authz revoke cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt /cosmos.gov.v1beta1.MsgVote --from alice
+```
+
+View authorization:
+```
+simd q authz grants cosmos1jxd2uhx0j6e59306jq3jfqs7rhs7cnhvey4lqh cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt /cosmos.gov.v1beta1.MsgVote
+```
+
+## 🎉 Congratulations 🎉
+
+By completing this tutorial, you have learned how to use the authz module.
+
+To learn more about the authorization module and different types of authorizations, check out the [authz module documentation](https://docs.cosmos.network/v0.43/modules/authz/).
```

Comment:
Link didn't work for me, not sure if it's already up?

Reply:
what is our version URL best practice? do we want to use master so the link automatically works? or we use the current version and then auto promote the topic for next version?

Reply:
It looks like there might be an issue with the deployment at the moment. :grimacing: This is the correct link. Also, I thought it might be best to use a specific version to prevent a potentially bad user experience if changes are made to master.